src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -0,0 +1,230 @@
+            </EuiText>
+          }
+        />
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiCard
+          style={{ width: '270px', height: '162px' }}
+          textAlign="left"
+          title={i18n.translate('workspace.initial.card.discoverInsights.title', {
+            defaultMessage: 'Discover insights',
+          })}
+          description={i18n.translate('workspace.initial.card.discoverInsights.description', {
+            defaultMessage: 'Explore data interactively to uncover insights.',
+          })}
+          footer={
+            <EuiText color="subdued" style={{ fontSize: '18px' }}>
+              {i18n.translate('workspace.initial.card.discoverInsights.footer', {
+                defaultMessage: 'with Discover',
+              })}
+            </EuiText>
+          }
+        />
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiCard
+          style={{ width: '270px', height: '162px' }}
+          title={''}
+          description={i18n.translate('workspace.initial.card.muchMore.description', {
+            defaultMessage: 'And much more...',
+          })}
+        />
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  const content = (
+    <EuiFlexGroup direction="column">
+      <EuiFlexItem grow={false}>
+        <EuiTitle size="l">
+          <h1>
+            {i18n.translate('workspace.initial.title', {
+              defaultMessage: 'Getting started with OpenSearch',
+            })}
+          </h1>
+        </EuiTitle>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} style={{ maxWidth: '650px' }}>
+        <EuiText grow={false}>
+          {i18n.translate('workspace.initial.description', {
+            defaultMessage:
+              'OpenSearch is a flexible, scalable, open-source way to build solutions for data-intensive search and analytics applications. Explore, enrich, and visualize your data, using developer-friendly tools and powerful integrations for machine learning, data processing, and more.',
+          })}
+        </EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} className="eui-displayInline">
+        <EuiButtonEmpty
+          href="https://www.opensearch.org/"
+          iconType="popout"
+          iconSide="right"
+          flush="left"
+          data-test-subj="workspace-initial-button-openSearch"
+        >
+          <EuiText>
+            {i18n.translate('workspace.initial.button.openSearch', {
+              defaultMessage: 'Learn more from documentation and more.',
+            })}
+          </EuiText>
+        </EuiButtonEmpty>
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false} style={{ maxWidth: '540px' }}>
+        <EuiPanel color="subdued" borderRadius="none" hasShadow={false} hasBorder={false}>
+          <EuiText>
+            <EuiIcon type="dashboardApp" size="l" />
+            &nbsp;&nbsp;Explore live demo environment at{' '}
+            <EuiLink href="https://future.playground.opensearch.org/">
```

Comment:
Is it OK to redirect to future playground?

Reply:
I will update the link